client/src/components/R&R/ReviewsBreakdown.jsx

```diff
@@ -5,11 +5,84 @@ import { ProductContext } from '../../context';
 
 const ReviewsBreakdown = () => {
   const { reviews, reviewsMeta, reviewSort, setReviewSort } = useContext(ProductContext);
+  const options = {
+    Size: ['A size too small', 'Perfect', 'A size too big'],
+    Width: ['Too narrow', 'Perfect', 'Too wide'],
+    Comfort: ['Uncomfortable', 'Ok', 'Perfect'],
+    Quality: ['Poor', 'What I expected', 'Perfect'],
+    Length: ['Runs short', 'Perfect', 'Runs long'],
+    Fit: ['Runs tight', 'Perfect', 'Runs long'],
+  };
+
+  const recommendPercentage = () => {
+    const total = Number(reviewsMeta.recommended.true) + Number(reviewsMeta.recommended.false);
+    return Math.round((reviewsMeta.recommended.true / total) * 100);
+  };
+
+  const characteristicArray = [];
+  const formatCharacteristics = () => {
+    Object.keys(reviewsMeta.characteristics).map((key) => {
+      const obj = { ...reviewsMeta.characteristics[key] };
+      obj.char = key;
+      characteristicArray.push(obj);
+    });
+    return characteristicArray;
+  };
```

Comment:
It's a bit strange to declare it outside and then return it within the function, especially since it's not a closure. I'd consider either passing it in--to increase the flexibility of the function--or declaring it inside, so it isn't polluting the file's global scope.